fw/http_parser.c

```diff
@@ -8415,25 +8548,40 @@ __h2_req_parse_if_msince(TfwHttpMsg *msg, unsigned char *data, size_t len,
 	if (r >= 0) {
 		parser->cstate.is_set = 1;
 		parser->cstate.if_msince_date = parser->_date;
-		h2_set_hdr_if_mod_since(req, &parser->cstate);
-
-		return CSTR_EQ;
+		return h2_set_hdr_if_mod_since(req, &parser->cstate);
 	}
 
 	return r;
 }
 
+void
+h2_set_hdr_pragma(TfwHttpReq *req, const TfwCachedHeaderState *cstate)
+{
+	if (cstate->is_set &&
+	    cstate->cache_ctl.flags & TFW_HTTP_CC_PRAGMA_NO_CACHE) {
```

Comment:
it'd better to place { on a new line to make the multi-line condition more readable